solidity/scripts/MultichainDeploy.sol

```diff
@@ -11,11 +11,11 @@ import {ScriptingLibrary} from './ScriptingLibrary/ScriptingLibrary.sol';
 contract MultichainDeploy is Script, ScriptingLibrary {
   uint256 public deployer = vm.envUint('DEPLOYER_PRIVATE_KEY');
   address constant CREATE2 = 0x4e59b44847b379578588920cA78FbF26c0B4956C;
-  string[] public chains = ['POLYGON_RPC', 'OPTIMISM_RPC', 'GOERLI_RPC'];
+  string[] public chains = ['MAINNET_RPC', 'POLYGON_RPC', 'OPTIMISM_RPC', 'ARBITRUM_RPC', 'GNOSIS_RPC', 'BSC_RPC'];
```

Comment:
Can you add the rpcs that are missing in the .env.example?

Reply:
Sure, Thanks for suggestion